pyproject.toml

```diff
@@ -1,42 +1,38 @@
 [build-system]
-requires = ["setuptools>=64.0.0", "setuptools-scm>1.5.4"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
```

Comment:
There is an equivalent `setuptools-scm` for `hatch` - `hatch-vcs`. I feel that going from automated versioning to manual versioning would be a downgrade and an extra overhead for the maintainers.

Reply:
that makes sense! We definitely want to avoid manual versioning. I will look into adding `hatch-vcs`

Reply:
Looks like this is referred to in the scientific packaging guide too https://learn.scientific-python.org/development/guides/packaging-simple/#versioning